draft-dejong-remotestorage-06.txt

```diff
@@ -522,8 +527,7 @@ Internet-Draft              remoteStorage                  November 2015
 
     Another example is "http://tools.ietf.org/html/rfc7233" with a
     string value of "GET" if Content-Range headers are supported for
-    GET requests as per [RANGE], "PUT" if they are supported for PUT
-    requests, and "GET,PUT" if supported for both.
+    GET requests as per [RANGE].
```

Comment:
if we don't define how range requests for PUT should work we should leave it out, as it will not be interoperable in any case...